.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
I think you can add this file to the `.gitignore`, seems like your editor specific config.

Reply:
I'm personally okay with having big `.gitignore` inside project. Don't see any problems with it :slightly_smiling_face:

Reply:
Although it looks all out of place, it was intentional.

[.editorconfig](https://editorconfig.org/) allows consistent styling between different editors and IDEs. It's an editor agnostic config file because it's supported by any editors.

I guess I'd rather remove this file from this PR unless you guys change your minds 😄